docs/database/updates/2018-06-09.sql

```diff
@@ -0,0 +1,4 @@
+-- Maybe backup login credentials if anything goes wrong
+-- CREATE TABLE users_bak LIKE users;
+-- INSERT users_bak SELECT * FROM users;
```

Comment:
Hmm so I actually forgot to uncomment this while deploying on the dev website... I still made a back up before running the shell command, with the altered password column. It's perhaps best to uncomment it by default? It doesn't cost much to have this backup. We just have to remember to clean it up later.

Reply:
Okay I see. The process of uncommenting is meant to avoid having the table being created without being noticed.